src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -128,6 +136,27 @@ public void testKeyEncryption() throws InvalidParametersException {
     }
 
     @Test
+    public void testAesdefaultmodepadding() throws InvalidParametersException {
+        String msg = "Test AES Default";
+        String decrptedMsg;
+        byte[] decrypted;
+        decrptedMsg = StringHelper.encrypt(msg);
+
+        try {
+            SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+            Cipher cipher = Cipher.getInstance("AES");
+            cipher.init(Cipher.DECRYPT_MODE, sks);
+            decrypted = cipher.doFinal(StringHelper.hexStringToByteArray(decrptedMsg));
+        } catch (NumberFormatException | IllegalBlockSizeException | BadPaddingException e) {
+            log.warning("Attempted to decrypt invalid ciphertext: " + decrptedMsg);
+            throw new InvalidParametersException(e);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+        assertEquals(msg, new String(decrypted));
```

Comment:
Add one or 2 more test strings just so that we are sure, including Strings that will be converted into a byte array that is > 128 bits. This is so we can check for any difference in mode of operations.